tests/i18n/urls_default_unprefixed.py

```diff
@@ -1,10 +1,12 @@
 from django.conf.urls import url
 from django.conf.urls.i18n import i18n_patterns
-from django.http import HttpResponse
+from django.http import HttpResponse, HttpResponseNotFound
 from django.utils.translation import ugettext_lazy as _
 
+
 urlpatterns = i18n_patterns(
     url(r'^(?P<arg>[\w-]+)-page', lambda request, **arg: HttpResponse(_("Yes"))),
     url(r'^simple/$', lambda r: HttpResponse(_("Yes"))),
+    url(r'^(?P<group1>.+)/(?P<group2>.+)/$', HttpResponseNotFound),
```

Comment:
Also keep it as HttpResponseNotFound as bug only occurs when that view throws 404.